src/System.Security.Cryptography.Algorithms/tests/ECDsaCngImportExportTests.cs

```diff
@@ -0,0 +1,72 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+using System.Security.Cryptography.EcDsa.Tests;
+
+namespace System.Security.Cryptography.Cng.Tests
+{
+    public class ECDsaCngImportExportTests : ECDsaTestsBase
+    {
+        [Fact]
+        public static void TestImportKey()
+        {
+            using (CngKey key = CngKey.Import(TestData.Key_ECDiffieHellmanP256, CngKeyBlobFormat.GenericPublicBlob))
```

Comment:
A) Why are we talking about CNG in the platform neutral test library?  B) Why are we talking about ECDH in an ECDSA test?